app/helpers/application_helper.rb

```diff
@@ -14,6 +14,115 @@ module ApplicationHelper
     ]
   }
 
+  CIVIL_URLS = [
```

Comment:
I would move all these code from `ApplicationHelper` and would create a new one, maybe called `CivilPartnershipHelper` - nothing should change implementation wise as helpers are a _global_ thing in rails

Reply:
One for divorce and one for civil partnerships? Or just create a tasklistHelper?

Reply:
One for both